src/setup/playwright.ts

```diff
@@ -18,14 +18,14 @@ export async function launchPlaywright(
     browser = await (
       await import("playwright")
     ).chromium.launchPersistentContext(tmpdir, {
+      ...(options.playwrightOptions ?? {}),
       headless: options.playwrightOptions?.headless,
       args: [
         `--disable-extensions-except=${metamaskPath}`,
         `--load-extension=${metamaskPath}`,
         ...(options.playwrightOptions?.args || []),
         `${options.puppeteerOptions?.headless ? "--headless=chrome" : ""}`,
-      ],
-      ...(options.playwrightOptions ?? {}),
+      ].filter((el) => el !== ""),
```

Comment:
maybe use destructing so there is no need for a filter?
 
`...(options.puppeteerOptions?.headless ? ["--headless=chrome"] : [])`